mlflow/protos/databricks_tracing.proto

```diff
@@ -39,6 +39,46 @@ service DatabricksTrackingService {
       rpc_doc_title: "Get Traces",
     };
   }
+
+  // =============================================================================
+  // Enablement RPCs
+  // =============================================================================
+
+  rpc createTraceUCStorageLocation (CreateTraceUCStorageLocation) returns (CreateTraceUCStorageLocation.Response) {
```

Comment:
@serena-ruan  The response here should be a `UCSchemaLocation` object directly. It will not be wrapped in a `Response` wrapper.

Reply:
E.g.:

```
  rpc createTraceUCStorageLocation(CreateTraceUCStorageLocation) returns (UCSchemaLocation) {

```